server.js

```diff
@@ -9,10 +9,10 @@ const bodyParser = require('body-parser');
 // This line sets the environment variables, since we are on our local machines
 // Therefore, in production (or whenever we are hosted on an actual server), this line can be removed
 //   along with the .env file
-require('dotenv').config();
-
-const db = require('./db/index');
+require('dotenv') // same as const dotenv = require('dotenv');
+.config(); // we just want to call .config, not save
```

Comment:
I like the commenting. Maybe indent ".config();" so we know it's chained to the previous line.

Reply:
Oh yeah.. Phil is totally right

Reply:
Makes sense to me.